docs/UserGuide.md

```diff
@@ -69,31 +69,210 @@ The following annotated text-boxes are used throught this guide to provide usefu
+
+![javaVersionLinux.jpg](images%2FjavaVersionLinux.jpg)
+This means that your machine is running Java version 17.0.11, which means that Java 17 is installed.
+
+---
+
+{: .alert .alert-info}
+:information_source: **Note**
+Depending on your system setup, you may need to install JavaFX separately. You can refer to the [OpenJFX installation guide](https://openjfx.io/openjfx-docs/#install-javafx) for detailed instructions.
+
+{: .alert .alert-info}
+:information_source: **Note**
+For users of other operating systems, as long as you have the following installed, you can use MediBase3:
+- Java/OpenJDK 17
+- JavaFX/OpenJFX 17
+
+{: .alert .alert-info}
+:information_source: **Note**
+For Linux users with Wayland compositors, ensure you install **OpenJFX 21** for better compatibility.
+
+
+---
+
+### Step 2: Download the MediBase3 JAR File
+
+1. Go to [this link](https://github.com/AY2425S1-CS2103T-F12-3/tp/releases/download/v1.3/medibase3.jar) and scroll down all the way to 'Assets'.  You should see something like this
+![githubAssets.png](images%2FgithubAssets.png)
+1. Click on `medibase3.jar` to download the latest MediBase3 JAR File.
+2. Save the file to a location of your choice (e.g., `Downloads` folder).
+
+---
+
+## Launching MediBase3
+
+1. Open **Command Prompt** (Windows) or **Terminal** (macOS).
+2. Navigate to the directory where the `.jar` file is located. For example:
+```
+cd Downloads
+```
+3. Run the application by typing:
+```
+   java -jar medibase3.jar
+   ```
+
+You should be greeted by the UI, which you will see in the next section.
+
+---
+
+## Overview of the GUI
+
+Once MediBase3 is running, you will see the main interface. Here’s an overview of the key parts:
+
+- **Command Box:** Where you can type and enter commands.
+- **Result Display:** Shows the result of the command you entered.
+- **Patient List:** Displays all the patients currently stored in the system.
+- **Appointment List:** Displays the scheduled appointments for patients.
+- **Menu (File/Help):** Provides additional options for managing the app (e.g., exit, access help).
+- **Data Storage Location Footer:** Displays the location where patient and appointment data are stored.
+
+![GUIOverview.jpg](images%2FGUIOverview.jpg)
+
+---
+
+## Overview of CLI
+
+MediBase3 can be controlled using commands in the **Command Line Interface (CLI)**.
+
+### CLI Command Structure
+
+Each command follows this structure:
+
+![CommandPromptFormat.png](images%2FCommandPromptFormat.png)
+
+Here’s a breakdown of the parts:
+
+| **Component** | **Description**                                    |
+|---------------|----------------------------------------------------|
+| **Command**   | The action you want to perform (e.g., `add`, `edit`) |
+| **Prefix**    | Identifies the data being entered (e.g., `n/` for name) |
+| **Parameter** | The value associated with the prefix (e.g., `John Doe`) |
+
+### Example:
+
+add n/John Doe p/98765432 e/johndoe@example.com
```

Comment:
Just a nit: would be better to use a proper example of `add`, else you can use a command with lesser parameters like `setPriority i/ !/`

Reply:
done